packages/web/src/js/FileUploader.ts

```diff
@@ -148,6 +148,15 @@ class FileUploader extends BaseComponent {
     }
   }
 
+  updateNameAttribute() {
+    // Removal of original name attribute for prevent of duplicate files from multiple inputs
```

Comment:
I'd just slightly polish the comment to be clearer:

```suggestion
    // Remove the original `name` attribute to prevent duplicate files from multiple inputs
```

Reply:
Of course